scripts/gitlab/build_and_test.sh

```diff
@@ -177,10 +177,11 @@ then
     if [[ "${truehostname}" == "corona" ]]
     then
         module unload rocm
+        module load cmake/3.23.1
```

Comment:
@adrienbernede RAJA now has min req version of cmake 3.23 for HIP builds and cmake 3.20 for all other builds. Is this the right way to make this change or is there a better way?

Reply:
There is a better way: you can grab the CMake executable used by Spack. Then, you just need to make sure that the Spack package is aware about the CMake requirement.

Reply:
Let me find the example code.

Reply:
https://github.com/LLNL/Umpire/blob/93b1441aaa258c1dcd211a552b75cff6461a2a8a/scripts/gitlab/build_and_test.sh#L130

Reply:
cmake_exe=`grep 'CMake executable' ${hostconfig_path} | cut -d ':' -f 2 | xargs`

Reply:
Thanks for the pointers. It seems to be working now. We can fine tune this later. I'm still learning where everything is.